web/views/task_view.ex

```diff
@@ -1,14 +1,16 @@
 defmodule CoursePlanner.TaskView do
   @moduledoc false
   use CoursePlanner.Web, :view
+  alias CoursePlanner.{SharedView, Volunteers}
+  alias Ecto.Changeset
 
-  def task_user(conn, task) do
-    if task.user_id do
-      link task.user.name, to: volunteer_path(conn, :show, task.user)
-    else
-      "no one"
-    end
+  def task_user(conn, volunteers) when not is_nil(volunteers) do
```

Comment:
is this shown properly? maybe it is worth to join the links with a comma or newline